Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:7
+
+ENV TZ Africa/Algiers
+
+RUN curl -Lo- https://yarnpkg.com/install.sh | bash
+ENV PATH $HOME/.yarn/bin:$PATH
+RUN mkdir $HOME/.yarn-cache
+VOLUME $HOME/.yarn-cache # TODO: best way?
```

Comment:
Do know either, maybe we could just ignore caching, yarn is fast enough for now. Unless you noticed issues about it